cli/src/commands/media/uploadVideo.ts

```diff
@@ -204,6 +204,7 @@ export default class UploadVideoCommand extends MediaCommandBase {
           'Content-Length': fileSize.toString(),
         },
         maxContentLength: MAX_FILE_SIZE,
+        maxBodyLength: MAX_FILE_SIZE,
```

Comment:
Adding reference to https://github.com/axios/axios/pull/2781
So we really only need to set the `maxBodyLength` here to MAX_FILE_SIZE
`maxContentLength` is the max length of the response which should never be this large after uploading.